lib/stat_tracker.rb

```diff
@@ -235,114 +224,26 @@ def winningest_team
     team_with_highest_win_percentage.team_name
   end
 
-  def biggest_bust(season_id)
-    preseason = []
-    reg_season = []
-    @games.each do |game| # could this be a method?
-      if season_id == game.season && game.type == "P"
-        preseason << game
-      elsif season_id == game.season && game.type == "R"
-        reg_season << game
-      end
+  def group_games_by_season_type(type, games = @games)
+    games.select do |game|
+      game.type == type
     end
+  end
+
+  def biggest_bust(season_id)
+    regular_season = group_games_by_season_type(("R"), games_by_season[season_id])
+    preseason = group_games_by_season_type(("P"), games_by_season[season_id])
```

Comment:
@allen4397 Hey Tim, why is the "P" between parenthesis here?

Reply:
Oh, I see what your saying. I think I was just tired and wasn't thinking. Looking at it now, it's a string argument so it should be fine without it. I can take it out.